catroid/src/main/res/layout/activity_recycler.xml

```diff
@@ -30,6 +30,21 @@
 
     <include layout="@layout/toolbar" />
 
+    <LinearLayout
```

Comment:
the fragment already has a progressbar, if you added the progressbar here because it is accessed somewhere, then we should remove the access (or move it into the fragment).